s3backuprestore/tagging.py

```diff
@@ -0,0 +1,442 @@
+import multiprocessing
+import queue
+import sys
+import threading
+import time
+from random import randint
+from datetime import datetime
+
+from .cw import put_metric
+from .log import logger
+
+
+class _CheckDeletedTag(threading.Thread):
+    def __init__(self, config, check_deleted_tag_queue, restore_queue,
+                 cw_metric_name='CheckDeletedTaggsErrors'):
+        """Checks if S3 objects are tagged as Deleted.
+
+        If objects are tagged as Deleted and its value is True, it would not
```

Comment:
Die Info sehe ich schon als wichtig an

Reply:
I updated the docstring. But to clarify this here as well. That means if the object has following TagSet: Key: Deleted, Value: True it will not be added to the queue.